rye/src/templates/LICENSE.txt

```diff
@@ -0,0 +1 @@
+{{ license_text }}
```

Comment:
I would add `.j2` to all templates as extension for syntax highlighting in editors.

Reply:
Sounds reasonable. I looked into the examples directory of minijinja as a reference and they didn't have `.j2`, but it makes sense to add it.